plugin/usage.go

```diff
@@ -23,8 +23,8 @@ var UsageFunc = func(c *cobra.Command) error {
 
 // CmdTemplate is the template for plugin commands.
 const CmdTemplate = `{{ bold "Usage:" }}{{if .Runnable}}
-  tanzu {{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}
-  tanzu {{.CommandPath}} [command]{{end}}{{if gt (len .Aliases) 0}}
+  tanzu{{ $target := index .Annotations "target" }}{{ if or (ne $target "global") (ne $target "") }} {{ $target }} {{ else }} {{ end }}{{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}
```

Comment:
Shouldn't it be `and` instead of `or` here? 
```suggestion
  tanzu{{ $target := index .Annotations "target" }}{{ if and (ne $target "global") (ne $target "") }} {{ $target }} {{ else }} {{ end }}{{.UseLine}}{{end}}{{if .HasAvailableSubCommands}}
```

Reply:
Yes. Good catch.  Updated